docs/source/noise/index.rst

```diff
@@ -110,60 +110,60 @@ Noise types for each column
   :header-rows: 1
 
   * - Column name
-    - Datasets present
+    - Applicable datasets
```

Comment:
I went ahead and changed this column name as discussed at the doc session last week. This is unrelated to the noise function order.